lib/rpmts.c

```diff
@@ -281,7 +281,7 @@ static int loadKeyringFromFiles(rpmts ts)
     int nkeys = 0;
 
     rpmlog(RPMLOG_DEBUG, "loading keyring from pubkeys in %s\n", pkpath);
-    if (rpmGlob(pkpath, NULL, &files)) {
+    if (rpmGlob(pkpath, RPMGLOB_NONE, NULL, &files)) {
```

Comment:
This seems like a counter-example to the above, i.e. here we most likely do want to detect the case of no matches (which `RPMGLOB_NOCHECK` would prevent).